frontend/src/services/Api.ts

```diff
@@ -122,13 +122,18 @@ export const login = async (
 
   const proxyUrl = getAPIProxy();
 
-  return axios({
+  const axiosRequest = {
     method: HTTP_VERBS.POST,
     url: proxyUrl ? `${proxyUrl}/${urls.authenticate}` : urls.authenticate,
     headers: getHeaders(proxyUrl),
-    auth: basicAuth(request.username, request.password),
     data: params
```

Comment:
Should we also do something similar for `token` and don't append `params` if token is unset?

Reply:
I'm fine minimizing the change.  Just noticed this.

Reply:
@jshaughn @jmazzitelli Yes I agree to minimize the change as it would be less disruptive.

Reply:
I just doing some manual testing right now. But on the face of it, it looks good. I'll be able to approve today some time if all looks good.